ldelf/ta_elf.c

```diff
@@ -1122,6 +1143,9 @@ static void load_main(struct ta_elf *elf)
 	save_symtab(elf);
 	close_handle(elf);
 	set_tls_offset(elf);
+	parse_property_segment(elf);
+	if (elf->bti == true)
```

Comment:
Please drop the ` == true` part, same below.

Reply:
ok